mangaki/mangaki/apps.py

```diff
@@ -0,0 +1,9 @@
+from django.apps import AppConfig
+
+
+class MangakiConfig(AppConfig):
+    name = 'mangaki'
+    verbose_name = 'Mangaki'
+
+    def ready(self):
+        from . import receivers
```

Comment:
Could you explain what does this line do in a comment? It is not that evident, I think.